packages/p2p/src/components/page-return/__tests__/page-return.spec.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import PageReturn from '../page-return.jsx';
+
+describe('<PageReturn />', () => {
+    it('Component should be rendered', () => {
+        render(<PageReturn />);
+
+        const el_dp2p_page_return_container = screen.getByTestId('dp2p-page-return_container');
+
+        expect(el_dp2p_page_return_container).toHaveClass('page-return');
+    });
+
+    it('Should render proper className passed from props', () => {
+        render(<PageReturn className={'test'} />);
+
+        const el_dp2p_page_return_container = screen.getByTestId('dp2p-page-return_container');
+
+        expect(el_dp2p_page_return_container).toHaveClass('page-return test');
+    });
```

Comment:
@artsemkavaleu-deriv IMHO we should not include 'page-return' className in your assertion because in your description of the test-case you only want to check **'test'** className, and also for example in future 'return-page' className can be removed, and your test-case fails

Reply:
sure, fixed, thx for notice!